README.rst

```diff
@@ -35,6 +35,10 @@ The ``gcovr`` command can produce different kinds of coverage reports:
      - |abstract-guide-html-details|
      - annotated source files
 
+   * - |abstract-option-template-dir|
```

Comment:
The `html-` is missing here. I'm not sure if this option shall be mentioned in the overview since it's only a customization for the `--html` or `--html-details`.
```suggestion
   * - |abstract-option-html-template-dir|
```